test/check-docker

```diff
@@ -176,14 +176,33 @@ class TestDocker(MachineCase):
         port = 3380;
         message = "HelloMessage."
 
+        # background: wait for the port to become available and echo message
+        # then start netcat server for port
+        # netcat has to run in the main process, otherwise docker might kill the container prematurely
+        # busybox netcat has reduced functionality (no --send-only or -c parameters)
+        # after 10 seconds without a port connection, the message "port not open" is printed instead
+        def echoCommand(listen_port, message_to_send):
```

Comment:
We need this for different messages, different ports and in the second check, we explicitly test setting the run command in the run container dialog.

Is it worth uploading a file into the vm to add a script?